tools/StandardAnchorTags/DiagnosticIDs.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace StandardAnchorTags;
+internal class DiagnosticIDs
+{
+    public const string TOC001 = nameof(TOC001);
+    public const string TOC002 = nameof(TOC002);
+}
```

Comment:
I'm not sure how useful this is, but it seems reasonable to sort errors and warnings over time.